DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/streaming_data_interfaces/sample.json

```diff
@@ -0,0 +1,71 @@
+{
+  "guid": "aeaca2ce-126e-452b-a6e3-2a3d5b1dbf55",
+  "name": "Streaming Interfaces",
+  "categories": ["Toolkit/oneAPI Direct Programming/C++SYCL FPGA/Tutorials/experimental/streaming_interfaces"],
+  "description": "An Intel® FPGA tutorial demonstrating how to use pipes to implement streaming interfaces on IP Components",
+  "toolchain": ["icpx"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["ide", "cmake"],
+  "languages": [{"cpp":{}}],
+  "commonFolder": {
+    "base": "../../..",
+    "include": [
+      "README.md",
+      "Tutorials/Features/experimental/streaming_interfaces",
+      "include"
+    ],
+    "exclude": []
+  },
+  "ciTests": {
+    "linux": [
+      {
+        "id": "fpga_emu",
+        "steps": [
+          "icpx --version",
+          "mkdir build",
+          "cd build",
+          "cmake ..",
+          "make fpga_emu",
+          "./threshold_packets.fpga_emu"
```

Comment:
threshold_packets? why not streaming_interfaces?

Reply:
Was the old name of the sample. Fixed now